File: solution/1_Fibonacci/1_Fibonacci/FibonacciTest.cs
using System;
using System.Text;
using System.Collections.Generic;
using System.Linq;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using fibonacci;

namespace test
{
    [TestClass]
    public class FibonacciTest
    {
        [TestMethod]
        public void fibonacci()
        {
            Assert.AreEqual(0, new Fibonacci().bereken(0));
            Assert.AreEqual(1, new Fibonacci().bereken(1));
            Assert.AreEqual(1, new Fibonacci().bereken(2));
            Assert.AreEqual(2, new Fibonacci().bereken(3));
            Assert.AreEqual(3, new Fibonacci().bereken(4));
            Assert.AreEqual(5, new Fibonacci().bereken(5));
            Assert.AreEqual(8, new Fibonacci().bereken(6));
            Assert.AreEqual(13, new Fibonacci().bereken(7));
            Assert.AreEqual(21, new Fibonacci().bereken(8));
            Assert.AreEqual(34, new Fibonacci().bereken(9));
            Assert.AreEqual(55, new Fibonacci().bereken(10));
            Assert.AreEqual(89, new Fibonacci().bereken(11));
            Assert.AreEqual(144, new Fibonacci().bereken(12));
            Assert.AreEqual(233, new Fibonacci().bereken(13));
            Assert.AreEqual(377, new Fibonacci().bereken(14));
            Assert.AreEqual(610, new Fibonacci().bereken(15));
            Assert.AreEqual(987, new Fibonacci().bereken(16));
            Assert.AreEqual(1597, new Fibonacci().bereken(17));
            Assert.AreEqual(2584, new Fibonacci().bereken(18));
            Assert.AreEqual(4181, new Fibonacci().bereken(19));
            Assert.AreEqual(6765, new Fibonacci().bereken(20));
        }
    }
}

File: workspace/1_Fibonacci/test/fibonacci/FibonacciTest.java
package fibonacci;

import static junit.framework.Assert.assertEquals;

import org.junit.Test;


public class FibonacciTest {
	
	@Test
	public void fibonacci() {
		assertEquals(0, new Fibonacci().calculate(0));
		assertEquals(1, new Fibonacci().calculate(1));
		assertEquals(1, new Fibonacci().calculate(2));
		assertEquals(2, new Fibonacci().calculate(3));
		assertEquals(3, new Fibonacci().calculate(4));
		assertEquals(5, new Fibonacci().calculate(5));
		assertEquals(8, new Fibonacci().calculate(6));
		assertEquals(13, new Fibonacci().calculate(7));
		assertEquals(21, new Fibonacci().calculate(8));
		assertEquals(34, new Fibonacci().calculate(9));
		assertEquals(55, new Fibonacci().calculate(10));
		assertEquals(89, new Fibonacci().calculate(11));
		assertEquals(144, new Fibonacci().calculate(12));
		assertEquals(233, new Fibonacci().calculate(13));
		assertEquals(377, new Fibonacci().calculate(14));
		assertEquals(610, new Fibonacci().calculate(15));
		assertEquals(987, new Fibonacci().calculate(16));
		assertEquals(1597, new Fibonacci().calculate(17));
		assertEquals(2584, new Fibonacci().calculate(18));
		assertEquals(4181, new Fibonacci().calculate(19));
		assertEquals(6765, new Fibonacci().calculate(20));
	}

}
